orttraining/orttraining/training_ops/cuda/collective/nccl_kernels.cc

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include "nccl_kernels.h"
+#include "orttraining/training_ops/cuda/collective/nccl_kernels.h"
```

Comment:
I think we should have UT covering those kernels, by mocking NCCL allreduce op for longer term.

Reply:
added.